synapse/lib/scrape.py

```diff
@@ -81,13 +92,49 @@ def cve_check(match: regex.Match):
     valu = s_chop.replaceUnicodeDashes(valu)
     return valu, cbfo
 
+# This excellent IPV6 regex comes from RFC3986. Slightly modified to not capture
+# '::' (any-address)
+ipaddr_define = r'''
+(?(DEFINE)
+  (?<OCTET>25[0-5]|2[0-4]\d|[01]?\d\d?)
+  (?<IPV4>(?:(?&OCTET)\.){3}(?&OCTET))
+  (?<H16>[0-9a-fA-F]{1,4})
+  (?<LS32>(?&H16):(?&H16)|(?&IPV4))
+  (?<IPV6>
+    (?:                                 (?:(?&H16):){6}(?&LS32)) |
+    (?:                              :: (?:(?&H16):){5}(?&LS32)) |
+    (?:(?:                 (?&H16))? :: (?:(?&H16):){4}(?&LS32)) |
+    (?:(?:(?:(?&H16):){0,1}(?&H16))? :: (?:(?&H16):){3}(?&LS32)) |
+    (?:(?:(?:(?&H16):){0,2}(?&H16))? :: (?:(?&H16):){2}(?&LS32)) |
+    (?:(?:(?:(?&H16):){0,3}(?&H16))? :: (?:(?&H16):){1}(?&LS32)) |
+    (?:(?:(?:(?&H16):){0,4}(?&H16))? ::                (?&LS32)) |
+    (?:(?:(?:(?&H16):){0,5}(?&H16))? ::                 (?&H16)) |
+    (?:(?:(?:(?&H16):){0,6}(?&H16))  ::                        )
+  )
+
+  (?<IPV4_ADDR>
+    (?<!\d|\d\.|[0-9a-fA-F:]:)
+    (?&IPV4)
+    (?!\d|\d\.|\.\d)
+  )
+
+  (?<IPV6_ADDR>
+    (?<![0-9a-fA-F]:|:[0-9a-fA-F]|::|\d\.)
+    (?&IPV6)
+    (?![0-9a-fA-F:]|\d\.|\.\d)
```

Comment:
also for this one is the `\d\.` covered by `[0-9a-fA-F:]`?

Reply:
Also not necessary.